README.md

```diff
@@ -17,24 +17,27 @@ A companion for Bézier path representation and geometry is [kurbo].
 
 ## Backends
 
+*For cross-platform use, the [`piet-common`][] crate reexports the most
```

Comment:
I don't think you need the following `[]`, but maybe I'm missing something.

Reply:
I've started doing this in some places; it is the way you're supposed to do implicit links per the original markdown 'spec'. 🤷